src/modules/home/home.user/OrderUser.tsx

```
@@ -93,56 +46,63 @@ export function UserOrders() {
+                      Total: ${order.total ? order.total.toFixed(2) : "N/A"}
                     </div>
                   </div>
                 </CollapsibleTrigger>
                 <CollapsibleContent>
                   <div className="mt-4 pl-4 border-l-2 border-muted">
                     <h4 className="font-medium mb-2">Products:</h4>
                     <ul className="space-y-2">
-                      {order.products.map((product, index) => (
-                        <li key={index} className="text-sm">
+                      {order.order_detail.map((item: any) => (
+                        <li className="text-sm">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)